modules/get-started/pages/quick-start.adoc

```diff
@@ -36,111 +55,343 @@ docker compose version
 // (step {"action":"wait", "duration": 10000})
 
-. Run the following in the directory where you saved the `docker-compose.yml` file:
+When the containers are running, you have:
+
+- A Redpanda cluster with a 30-day free trial license.
+- A running data streaming pipeline generating data into the `logins` and `transactions` topics.
+- A Wasm data transform processing data from `logins` into `edu-filtered-domains`.
+- A secured instance of Redpanda Console with login authentication.
+
+== Explore Redpanda Console
+
+Redpanda Console is a developer-friendly web UI for managing and debugging your Redpanda cluster and your applications. This section provides practical examples and scenarios to help you understand how to leverage Redpanda Console for different use cases, including data observability, Redpanda management, access control, and connectivity.
+
+Redpanda Console was deployed as part of the `docker-compose.yml` file and is running locally on port 8080.
+
+=== Log in to Redpanda Console
+
+Redpanda Console is configured with plain login credentials (enterprise feature) for an admin user called `jane`. To start using Redpanda Console, you first need to log in with the credentials defined in the `docker-compose.yml` file.
+
+. Open your web browser and go to http://localhost:8080/login.
+. Enter the following credentials:
 +
-```bash
-docker compose up -d
-```
+--
+- Username: `jane`
+- Password: `some-other-secret-password`
+--
 +
-[.no-copy]
+This user is an admin user that has access to all features, including the Admin panel.
+
+You should now see an overview of your cluster's status, health, and broker details.
+
+image::console:overview.png[]
+
+// (step {"action":"checkLink", "url": "http://localhost:8080"})
+// (step {"action":"goTo", "url": "http://localhost:8080/login"})
+// (step {"action":"find", "selector": "[data-testid='auth-username-input']", "typeKeys": "jane", "click": true})
+// (step {"action":"find", "selector": "[data-testid='auth-password-input']", "typeKeys": "some-other-secret-password", "click": true})
+// (step {"action":"find", "selector": "[data-testid='auth-submit']", "click": true})
+// (step {"action":"wait", "duration": 15000})
+// (step {"action":"saveScreenshot", "path": "overview.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+
+To view details about a specific broker, click *View* at the end of the row in the *Broker Details* table.
+
+// (step {"action":"find", "selector": ".chakra-table tbody tr:last-child td:last-child button", "matchText": "View", "click": true})
+// (step {"action":"wait", "duration": 15000})
+// (step {"action":"saveScreenshot", "path": "broker-overview.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+
+image::console:broker-overview.png[]
+
+See also: xref:console:config/security/plain.adoc[].
```

Comment:
@JakeSCahill just noticed that some of these **See also:** lines include final punctuation, and some don't. I don't think we have a standard for them, but all of them on the same page should be consistent.